src/Microsoft.ApplicationInsights.AspNet/TelemetryInitializers/ComponentVersionTelemetryInitializer.cs

```diff
@@ -1,42 +1,45 @@
 ﻿namespace Microsoft.ApplicationInsights.AspNet.TelemetryInitializers
 {
     using System;
-    using System.Collections.Generic;
-    using System.Linq;
-    using System.Threading.Tasks;
     using Channel;
     using DataContracts;
     using Microsoft.AspNet.Http;
     using Microsoft.Extensions.Configuration;
-    using Microsoft.ApplicationInsights.Extensibility;
 
     /// <summary>
     /// A telemetry initializer that populates telemetry.Context.Component.Version to the value read from project.json
     /// </summary>
-    public class ComponentVersionTelemetryInitializer : ITelemetryInitializer
+    public class ComponentVersionTelemetryInitializer : TelemetryInitializerBase
```

Comment:
You don't appear to use any extra functionality from TelemetryInitializerBase so you can stick with just the interface ITelemetryInitializer.